src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,87 @@
+:ruby
+  @pagetitle = "Tasks for #{User.current}"
+  @layouttype = 'custom'
+  @crumb_list = [@pagetitle]
+
+.container
+  .card#reviews
+    .bg-light
+      %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+        %li.nav-item
+          - cache "#{User.current.cache_key}-reviews-in" do
+            %a{ href: '#reviews-in', title: "Requests that #{User.current.login} has to review", class: 'nav-link text-nowrap active' }
+              Incoming Reviews (#{User.current.involved_reviews.count})
+
+    .card-body
+      .tab-content#reviews-in
+        = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: user_requests_path(User.current) })
+  .card.mt-3#requests
+    .bg-light
+      %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+        %li.nav-item
+          - cache "#{User.current.cache_key}-requests-in" do
+            %a#requests-in-tab{ href: '#requests-in', title: "Requests that #{User.current.login} has to merge",
```

Comment:
same here and in some other places...

Reply:
We have decided to keep all of them as `"#{User.current}"` since `User#to_s` is calling `User#login` and not use `User#name` for the moment.